bootstrap.sh

```diff
@@ -96,48 +98,51 @@ else
         if [ -x "$(command -v apt)" ]; then
             DESIRED_BOOST_VERSION=${BOOST_VERSION//_/.}
             # install missing packages if necessary
-            boostlibnames=("libboost-system" "libboost-filesystem" 
+            boostlibnames=("libboost-system" "libboost-filesystem"
                     "libboost-graph" "libboost-program-options"
                     "libboost-log" "libboost-thread")
             additional_boost_libs=()
             for boost_lib in ${boostlibnames[@]}; do
-                dpkg -s "$boost_lib${DESIRED_BOOST_VERSION}" >/dev/null 2>&1 || additional_boost_libs+=("$boost_lib${DESIRED_BOOST_VERSION}")
+                dpkg -s "$boost_lib${DESIRED_BOOST_VERSION}" >/dev/null 2>&1 ||
+                dpkg -s "$boost_lib${DESIRED_BOOST_VERSION}.0" >/dev/null 2>&1 ||
+                additional_boost_libs+=("$boost_lib${DESIRED_BOOST_VERSION}") ||
+                additional_boost_libs+=("$boost_lib${DESIRED_BOOST_VERSION}.0")
```

Comment:
Why do we need to explicitly add a `.0` at the end here?

Reply:
Hello @pdschubert ,  
Thank you for reviewing the changes. 

If an installed version of the boost library version is `1.73.0` then the current check fails to detect as it compares 1.73 with 1.73`.0` and the script exits. Considering the boost releases on https://www.boost.org/users/history/ , the releases are in a pattern of 1.xy.0. So, adding a `.0` check works fine.